src/OrchardCore/OrchardCore.Infrastructure.Abstractions/Settings/SiteServiceExtensions.cs

```diff
@@ -0,0 +1,11 @@
+using System.Threading.Tasks;
+
+namespace OrchardCore.Settings;
+
+public static class SiteServiceExtensions
+{
+    public static async Task<T> GetSettingsAsync<T>(this ISiteService siteService) where T : new()
+    {
+        return (await siteService.GetSiteSettingsAsync()).As<T>();
+    }
```

Comment:
```suggestion
    public static async Task<T> GetSettingsAsync<TSettings>(this ISiteService siteService) where TSettings : new()
        => (await siteService.GetSiteSettingsAsync()).As<TSettings>();
```

Reply:
Many people including @sebastienros and @Piedone do not like expression bodied methods. So I am defaulting to not use them any more.

Reply:
Not sure. This is just how VS formats the code and I keep it as is

Reply:
> Many people including @sebastienros and @Piedone do not like expression bodied methods. So I am defaulting to not use them any more.

It's not my or anyone else likes or dislikes, we should use a language feature whenever it's possible and fits

> Not sure. This is just how VS formats the code and I keep it as is

Yesterday you told me the VS doesn't add spaces when you move to second line, I need to check this myself

Reply:
Feel free to try the change in VS :) 

I would suggest opening a discussion about formatting not in this PR. The requested change does not add a value to the PR other than adhere to a personal preference. I don't mind method bodied methods. But, others seem to mind them. Opening a separate discussion would be a good start.

Reply:
Oh no, I actually like them, and at Lombiq we enforce them with code analysis too. It's just that in a prior exchange under a PR I pointed out, mistakenly, that we consciously don't use them in OC, but it turned out we actually do. Since then, I'm not disputing using them.

Also see to avoid such discussions in the future: https://github.com/OrchardCMS/OrchardCore/issues/7950#issuecomment-2138287937

Reply:
no problem here. I used expression bodied where possible.